### DISCUSSION TITLE
I have trouble with my web link

### DISCUSSION THREAD

**Discussion by Abareyellow** [community/community#22934] [Other Feature Feedback, Questions, & Ideas]
<p>I am having trouble with the web link to my website. It comes up with a 404 error. I know this probably been but I have tried everything and there has been no success. </p>
<p> web link: <a href="https://abareyellow.github.io/VoiceActors/" rel="nofollow noopener">https://abareyellow.github.io/VoiceActors/</a></p>
<p>Repository : <a href="https://github.com/Abareyellow/VoiceActors" rel="nofollow noopener">https://github.com/Abareyellow/VoiceActors </a></p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/abareyellow">@abareyellow</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the Pages board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/abareyellow">@abareyellow</a>,</p>
<p>It looks like your link is currently working. Were you able to resolve your issue?</p>
<p>Thanks!</p>

**Reply by cbates8979** [reply]
Can u help me out my TronLink I can't transfer or see balance and I've activated to no avail not even discord could help me. I'm stuck in watch mode and can see a partial balance I just need out of watch mode I tried putting in mnemonics again to no avail.

**Comment by ulisigi** [comment, marked as answer]
<p>Hi <a class="mention" href="/abareyellow">@abareyellow</a>,</p>
<p>I’ve found that it can sometimes take 30 minutes or more for Pages to deploy our work. The first time I experimented with it, I had to add /index.html to the end of the supplied link to get around the 404. By the next morning, however, it worked properly. As <a class="mention" href="/that-pat">@that-pat</a> mentioned, your link seems to be working fine now.</p>
<p>Happy coding!</p>
